synapse/lib/stormlib/gen.py

```diff
@@ -136,6 +136,34 @@ class LibGen(s_stormtypes.Lib):
                       {'name': 'name', 'type': 'str', 'desc': 'The name of the place.'},
                   ),
                   'returns': {'type': 'node', 'desc': 'A geo:place node with the given name.'}}},
+        {'name': 'fileBytesBySha256',
```

Comment:
Was it intentional to leave these as functions-only without corresponding commands?

Reply:
Yep, visi said he didn't want to add more commands because of feedback that there were already too many commands.